src/Compilers/Core/MSBuildTask/DotnetHost.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace Microsoft.CodeAnalysis.BuildTasks
+{
+    class DotnetHost
```

Comment:
Both, I guess? For example, this file https://github.com/dotnet/cli/blob/master/test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/DotnetCommand.cs#L8 uses the namespace `DotNet`, but the class name is `Dotnet`.